rust/connlib/tunnel/src/gateway/flow_tracker.rs

```diff
@@ -33,6 +33,15 @@ pub struct FlowTracker {
     created_at_utc: DateTime<Utc>,
 }
 
+/// Additional properties we track for a client.
+#[derive(Debug, Clone, Default)]
+pub struct ClientProperties {
+    pub version: String,
+    pub device_serial: String,
```

Comment:
```suggestion
    pub device_uuid: Option<String>,
    pub firebase_installation_id: Option<String>,
    pub identifier_for_vendor: Option<String>,
    pub device_serial: Option<String>,
```

for more see https://developer.apple.com/documentation/uikit/uidevice/identifierforvendor